website/docs/r/workspaces_directory.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
This needs a reference to the resource documentation in the file https://github.com/terraform-providers/terraform-provider-aws/blob/master/website/aws.erb